srcpkgs/ProcDump-for-Linux/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'ProcDump-for-Linux'
+pkgname=ProcDump-for-Linux
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+makedepends="zlib-devel"
+depends="gdb"
+short_desc="Provides applications developers to create core dumps"
+maintainer="<ndowens@yahoo.com>"
+license="MIT"
+homepage="https://github.com/Microsoft/ProcDump-for-Linux"
+distfiles="https://github.com/Microsoft/${pkgname}/archive/${version}.tar.gz"
+checksum="284b92a83ea7763b3edb5a300b8ef97c2c2a4a69c2efaca4822423b71c605a14"
```

Comment:
doesn't need to be quoted

Reply:
Any of the template or just those parts?